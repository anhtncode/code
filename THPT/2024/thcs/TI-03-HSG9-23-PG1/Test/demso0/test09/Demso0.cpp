 #include <bits/stdc++.h>
#define ll long long
using namespace std;

ll t, n, x, d;

int main ()
{
	ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    freopen("DEMSO0.inp","r",stdin);
    freopen("DEMSO0.out","w",stdout);
    cin>>t;
    while (t--)
    {
        cin>>n;
    	d=0, x=5;
    	while (x <= n)
    	{
    		d += n/x;
    		x *= 5;
    	}
    	cout<<d<<endl;
    }
	return 0;
}
