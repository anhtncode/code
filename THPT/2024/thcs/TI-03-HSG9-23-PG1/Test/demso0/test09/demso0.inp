18
19
30
42
63
65
110
135
333
19
30
42
63
65
110
135
333
