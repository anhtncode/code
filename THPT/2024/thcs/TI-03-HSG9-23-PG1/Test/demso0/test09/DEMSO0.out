52
56
83
113
139
150
207
232
474
506
833
138
1637
1775
2281
